exchanges.yml

```diff
@@ -261,24 +261,15 @@ components:
       type: object
       description: Object containing information about the exchange to be created.
       properties:
-        exchangeInformation:
+        expires:
+          type: string
+          description: The date and time (as specified by ISO 8601) that the exchange expires.
```

Comment:
This should be XML Schema dateTimeStamp (due to years of debate in the VCWG about timezone requirements).